compiler/include/stmt.h

```diff
@@ -317,6 +317,9 @@ struct Witnesses {
 
 class ImplementsStmt final : public Stmt {
 public:
+  static ImplementsStmt* build(InterfaceSymbol* isym,
+                               CallExpr* actuals,
+                               BlockStmt* body);
   static ImplementsStmt* build(const char* name,
```

Comment:
do we still need the `const char*` overload? Is it ever the correct thing to use?

 It is still used by `convert-uast`, but I suspect that to make `R implements Other.I` work that also should change.

Reply:
Well, we don't always have scope resolution results available in `convert-uast.cpp`, so `const char*` will be needed sometimes. I can adjust it to use the interface symbol when possible.